tests/metrics/estimate_gas.py

```diff
@@ -652,18 +636,11 @@ def estimate_gas(analyzer: AnalyzeGas = None) -> None:
     tx = staker_functions.confirmActivity().transact({'from': ursula1})
     testerchain.wait_for_receipt(tx)
     testerchain.time_travel(periods=1)
-    # Deploy adjudicator to estimate slashing method in StakingEscrow contract
-    adjudicator, _ = testerchain.deploy_contract(
-        'Adjudicator', staking_agent.contract.address, ALGORITHM_SHA256, MIN_ALLOWED_LOCKED - 1, 0, 2, 2
-    )
-    tx = staker_functions.setAdjudicator(adjudicator.address).transact()
-    testerchain.wait_for_receipt(tx)
-    adjudicator_functions = adjudicator.functions
```

Comment:
Need some way to deploy `Adjudicator` with specific parameters because `Adjudicator` from agent has other parameters that made this tests cases non-working.

Reply:
Well, they work right now...I didn't change anything in the tests. The function `TesterBlockchain.bootstrap_network()`, which is currently used by this script, deploys all the contracts by default.

Reply:
I mean not working properly: here specific (mostly worst) cases when slashing multiple sub stakes. And this achieved by specific parameters. So now it's running but I assume that always estimates only default slashing of one sub stake

Reply:
Well, if you want to use non-default parameters for the deployed contracts, then we should't use `TesterBlockchain.bootstrap_network()` at the start of this test. In any case, I think that change falls out of the scope of this PR.

Reply:
I agree that this is out of scope, maybe return this dirty hack (to make this estimation correct) and open an issue?

Reply:
Opened #1104